cloud_with_stamp: true
min_range: 0.5
max_range: 150.0